subsys/bluetooth/controller/Kconfig

```diff
@@ -16,26 +16,27 @@ if BT_CTLR
 
 choice BT_LL_CHOICE
 	prompt "Bluetooth Link Layer Selection"
-	default BT_LL_SW
 	help
 	  Select the Bluetooth Link Layer to compile.
 
-config BT_LL_SW_LEGACY
+config BT_LL_SW_SPLIT
```

Comment:
You need to rename this to `BT_LL_SW`, that's what we agreed with @joerchan right? no more "split" in the name

Reply:
Using the same BT_LL_SW will make users’ config file be valid after switching to new LL but without them noticing the change. Users should know the change in design. I will keep the _SPLIT until legacy is deleted.

Reply:
That's really not the case if they are just using `CONFIG_BT` on a Nordic board, right? `BT_LL_SW_SPLIT` will be automatically enabled without them noticing

Reply:
Those who dont use explicit BT_LL_SW dont care if the macro is called BT_LL_SW_SPLIT or anything. We get them to use the new LL. Those who did use (for any reason) BT_LL_SW will now know that it is deprecated and take an informed decision to use BT_LL_SW_SPLIT.